### PR TITLE
Deleting "invisible" cookie should be no-op action

### PR DESCRIPTION
**Pull request by alex-savchuk**
Deleting "invisible" cookie should do nothing (be no-op action) according to 
https://code.google.com/p/selenium/wiki/JsonWireProtocol#/session/:sessionId/cookie/:name

It fixes https://code.google.com/p/selenium/issues/detail?id=5187

### CODE REVIEW

File: src/com/opera/core/systems/OperaDriver.java [hunk 1]
```diff
@@ -820,7 +820,11 @@ public void addCookie(Cookie cookie) {
 
     public void deleteCookieNamed(String name) {
       Cookie cookie = getCookieNamed(name);
-      deleteCookie(cookie);
+      if (cookie != null) {
+        deleteCookie(cookie);
+      } else {
```
Comment (andreastt) [inline src/com/opera/core/systems/OperaDriver.java]: Unnecessary else block; please fix and I'll integrate (-:

Comment (alex-savchuk) [inline src/com/opera/core/systems/OperaDriver.java]: ok, i'll fix asap.

**Comment by alex-savchuk** [comment]
Unnecessary else branch is removed.

**Comment by andreastt** [comment]
Thanks!  Integrated in e5ec9119de329f3ffc55fd24d86f9b6f8b8741f2.

**Pull request closed without merging**